java/README.md

```diff
@@ -0,0 +1,308 @@
+This directory contains the java worker, with the following components.
+
+* java/api: Ray API definition
+* java/common: utilities
+* java/hook: binary rewrite of the Java byte-code for remote execution
+* java/runtime-common: common implementation of the runtime in worker
+* java/runtime-dev: a pure-java mock implementation of the runtime for fast development
+* java/runtime-native: a native implementation of the runtime
+* java/test: various tests
+* src/local_scheduler/lib/java: JNI client library for local scheduler
+* src/plasma/lib/java: JNI client library for plasma storage
+
+# Build and test
+
+```sh
+# build native components
+sh ../build.sh -l java
```

Comment:
I had to replace this with `../build.sh -l java` to make it work. With `sh` I got `../build.sh: 10: ../build.sh: Syntax error: "(" unexpected`.

Reply:
I will fix it soon, thanks.